opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -70,9 +70,14 @@ def shutdown(self) -> None:
         """Called when a :class:`opentelemetry.sdk.trace.Tracer` is shutdown.
         """
 
-    def force_flush(self) -> None:
+    def force_flush(self, timeout_millis: int = 30000):
         """Export all ended spans to the configured Exporter that have not
         yet been exported.
+
+        An exception is raised if the timeout is exceeeded.
```

Comment:
(continuing from https://github.com/open-telemetry/opentelemetry-python/pull/397#discussion_r375583947)

I think an exception is kinda OK if we use a default timeout, but I would prefer a boolean if we had a default timeout of infinite. I am slightly in favor of making Infinite the default.

But consider https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/error-handling.md#basic-error-handling-principles, which says that we should rather not throw exceptions.